main.go

```diff
@@ -21,14 +21,15 @@ import (
 )
 
 type config struct {
-	Once       bool          `envconfig:"GC_ONCE"`
-	Debug      bool          `envconfig:"GC_DEBUG"`
-	Color      bool          `envconfig:"GC_DEBUG_COLOR"`
-	Pretty     bool          `envconfig:"GC_DEBUG_PRETTY"`
-	Images     []string      `envconfig:"GC_IGNORE_IMAGES"`
-	Containers []string      `envconfig:"GC_IGNORE_CONTAINERS"`
-	Interval   time.Duration `envconfig:"GC_INTERVAL" default:"5m"`
-	Cache      string        `envconfig:"GC_CACHE" default:"5gb"`
+	Once       	bool          `envconfig:"GC_ONCE"`
+	Debug      	bool          `envconfig:"GC_DEBUG"`
+	Color      	bool          `envconfig:"GC_DEBUG_COLOR"`
+	Pretty     	bool          `envconfig:"GC_DEBUG_PRETTY"`
+	Images     	[]string      `envconfig:"GC_IGNORE_IMAGES"`
+	Containers 	[]string      `envconfig:"GC_IGNORE_CONTAINERS"`
+	Interval    time.Duration `envconfig:"GC_INTERVAL" default:"5m"`
```

Comment:
This looks wrong formatted

Reply:
Yes sorry, I now `gofmt`ed all my changes